namespace py hello

service HelloService {
  string sayHello(1:string name)
}
